cli/uninstall/uninstall.go

```diff
@@ -112,12 +112,30 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 			return err
 		}
 	}
+	// We need to try to delete all versions with 'v' prefix before x.y.z version
+	// and without it.
+	versionsToDelete := make([]string, 0, 2)
+	versionsToDelete = append(versionsToDelete, programVersion)
+
+	if program == search.ProgramTt &&
+		strings.Contains(programVersion, ".") && programVersion[0] != 'v' {
```

Comment:
```suggestion
		programVersion != 'master' && programVersion[0] != 'v' {
```
The `tt install` logic does not contain such checks.

Reply:
What if we have `tt uninstall tt <hash>`? The `programVersion` won't be master and also won't contains 'v' prefix, so it will be `v<hash>` and program won't be uninstalled as it was expected. That is why I want to determine if `programVersion` satisfies the `x.y.z`.

Reply:
Than we need the same for `tt install` too. The idea is keep the logic in sync for install and uninstall.